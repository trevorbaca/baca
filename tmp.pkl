ccopy_reg
_reconstructor
p1
(c__main__
Foo
p2
c__builtin__
object
p3
NtRp4
(dp5
S'x'
S'red'
p6
sb.