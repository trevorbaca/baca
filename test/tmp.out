		\beam #0 #3 c'32 [
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #0 c'32 ]
	\times 4/7 {
		\beam #0 #3 c'32 [
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #0 c'32 ]
	}
	\times 4/7 {
		\beam #0 #3 c'32 [
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #3 c'32
		\beam #3 #0 c'32 ]
	}